README.md

```diff
@@ -20,3 +20,55 @@ Suzaku Core contracts are inspired by [Symbiotic Core contracts](https://github.
 | ---------------------------------------------------------- | ---------------------------------------------------------------------------------------------------- |
 | [L1Registry](./src/contracts/L1Registry.sol)               | [NetworkRegistry](https://github.com/symbioticfi/core/blob/main/src/contracts/NetworkRegistry.sol)   |
 | [OperatorRegistry](./src/interfaces/IOperatorRegistry.sol) | [OperatorRegistry](https://github.com/symbioticfi/core/blob/main/src/contracts/OperatorRegistry.sol) |
+
```

Comment:
I would move this section to `src/collateral`

Also, you can inspire on this. https://docs.suzaku.network/suzaku-protocol/for-builders/collateral-class

Reference to symbiotic should be for acknowledgment and audits.